inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -720,17 +705,23 @@
             },
             "type": "object"
         },
-        "thesaurus_terms": {
+        "energy_ranges": {
+            "description": "Ranges of energies the record refers to",
+            "items": {
+                "minimum": 0,
+                "type": "integer"
+            },
+            "type": "array",
+            "uniqueItems": true
```

Comment:
Hmmm, this is actually not true, just deleting the `uniqueItems` is enough? or should I set it to `false`?

Also, is there a way to actually test this? (maybe it's part of the integration tests, not sure)